static/js/components/elements/icons/arrow_back.tsx

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * Material Icon: Arrow Back
+ * Source: https://fonts.google.com/icons
+ */
+
+import React, { ReactElement } from "react";
+
+export const ArrowBack = (
+  props: React.SVGProps<SVGSVGElement>
+): ReactElement => (
+  <svg
```

Comment:
Would it make sense to load the vector from a flat .svg file here? Maybe then both the client & server side could use the same directory of svg files (not sure how easy that is with the current folder structure).

Reply:
Thanks @nick-next -that makes sense! The script & embedding svgs directly sounds good to me